blocks/featured/featured.css

```diff
@@ -47,20 +50,49 @@
   font-size: var(--body-font-size-m);
 }
 
+.featured.plain .featured-recipe span {
+  display: block;
+  font-family: var(--heading-font-family);
+  font-size: var(--heading-font-size-l);
+  color: var(--text-color);
+}
+
+.featured .featured-recipe.button-container a {
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%,-50%);
+  padding: 1.25rem;
+  height: auto;
+  font-size: 18px;
```

Comment:
I think this could come from the global styles - IIRC, I do set button to be 18px in global now in one of the last PRs

Reply:
ok, I'll keep that in mind and once we have everything merged check if we can remove it